articles/connections/database/custom-db/templates/login.md

```diff
@@ -470,55 +455,41 @@ function login(email, password, callback) {
 
 ```sql
 function login(email, password, callback) {
-  var oracledb = require('oracledb');
+  const bcrypt = require('bcrypt');
+  const oracledb = require('oracledb');
   oracledb.outFormat = oracledb.OBJECT;
 
   oracledb.getConnection({
       user: configuration.dbUser,
       password: configuration.dbUserPassword,
-      connectString: "CONNECTION_STRING" // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
+      connectString: 'CONNECTION_STRING' // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
     },
     function(err, connection) {
-      if (err) {
-        return callback(err);
-      }
-      connection.execute(
-        "SELECT ID, EMAIL, PASSWORD, EMAIL_VERIFIED, NICKNAME" +
-        " FROM Users " +
-        " WHERE EMAIL = :email", [email],
-        function(err, result) {
-          if (err) {
-            doRelease(connection);
-            return callback(new Error(err));
-          }
-          doRelease(connection);
-          if (result.rows.length === 0) {
-            return callback(null);
-          }
-          bcrypt.compare(password, result.rows[0].PASSWORD, function(err, isValid) {
-            if (err) {
-              return callback(new Error(err));
-            } else if (!isValid) {
-              callback(new WrongUsernameOrPasswordError(email));
-            } else {
-              var userProfile = {
-                user_id: result.rows[0].ID,
-                nickname: result.rows[0].NICKNAME,
-                email: result.rows[0].EMAIL,
-                email_verified: result.rows[0].EMAIL_VERIFIED
-              };
-              callback(null, userProfile);
-            }
-          });
+      if (err) return callback(err);
+
+      const query = 'select ID, EMAIL, PASSWORD, NICKNAME from Users where EMAIL = :email';
```

Comment:
In Heroku review, code block renders strangely (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/login#oracle